### PR TITLE
Smoke test: also test deploy alerting module with terraform

### PR DESCRIPTION
**Pull request by chaodaiG**
/hold
The test hasn't pass yet

### CODE REVIEW

**Review by sethvargo — APPROVED** [review]
/lgtm

**Comment by sethvargo** [comment]
@chaodaiG do you still want to get this in?

**Comment by chaodaiG** [comment]
Haven't got much luck making this succeed.  @yegle, can we reliably deploy alerting module without manual intervention?

**Comment by yegle** [comment]
@chaodaiG Can you merge main into your PR so I can see what's failing?

**Comment by chaodaiG** [comment]
@yegle , I just rebased from main, and it actually succeeded in my local test!

File: terraform-e2e-ci/main.tf [hunk 1]
```diff
@@ -55,6 +55,17 @@ module "en" {
...
 
+module "en-alerting" {
+  source = "../terraform/alerting"
+
+  monitoring-host-project     = var.project
+  verification-server-project = var.project
+
+  adminapi_hosts  = ["adminapi.notexistdomain.notexist"]
+  apiserver_hosts = ["apiserver.notexistdomain.notexist"]
+  server_hosts    = ["encv-test.notexistdomain.notexist"]
```
Comment (yegle) [inline terraform-e2e-ci/main.tf:66]: Add `depends_on = [module.en]` here.

**Comment by yegle** [comment]
/lgtm

Please see my comment above.

**Pull request merged**